testplan/docs/kubera-e2e-release-1.12.md

```diff
@@ -0,0 +1,54 @@
+---
+id: kubera-e2e-release-1-12
+title: Kubera E2E Release 1.12
+sidebar_label: Kubera E2E 1.12
+---
+
+## RELEASE NOTES
+This is the third release of Kubera end to end (E2E) pipelines. Team continued their focus on stabilising the pipelines as well as verifying new Kubera features. Needless to say team was successful in finding numerous bugs given the state of flux that Kubera is in today. In this release, KONVOY E2E pipeline joined the ranks of being considered as a stable pipeline alongside AWS E2E pipeline. This is significant considering the fact that KONVOY pipeline is an on premise cluster that is setup on VMware VMs. It takes into account VM snapshots to bring up the K8s clusters really fast. KONVOY E2E pipeline tries its best to match with AWS pipeline that is blessed with better networking speed since later is running on cloud. A big shoutout to @Aman Gupta and @giri to make this a reality.
+
+### Rancher E2E Pipeline
+In addition to AWS and KONVOY, E2E uses Rancher as a Kubernetes platform to run end to end test cases. Rancher pipeline still continues to face a lot of issues. This has led to either its pipeline runs getting cancelled or marked as failed. In this release, it was mostly due to frequent reboots of underlying VMware VMs. As of today, these VMs have their motherboard replaced and things have started to look fine. Team will focus diligently to elevate Rancher as a stable pipeline in upcoming release.
```

Comment:
it was mostly due to frequent reboots of Esx.... As of today, the motherboard of this ESX had been replaced.